components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -440,6 +438,15 @@ public Timestamp getTimeStamp(String key, String type) {
             } catch (IdentityException e) {
                 //ignore
                 log.error("Error while obtaining the database connection", e);
+            } finally {
+                if(resultSet != null){
+                    try{
+                        resultSet.close();
+                    } catch (SQLException e){
+                        log.error("Error when closing the result set of session time stamps");
```

Comment:
This method is changed after this PR : https://github.com/wattale/carbon-identity/commit/cf718b67f8853fb35c3dce53747e918475d35879. Therefore the wrong error log is no longer exists.